assets/app/styles/_messages.less

```diff
@@ -160,27 +164,20 @@ ul.messenger-theme-flat .messenger-message .messenger-actions a {
 ul.messenger-theme-flat {
   .messenger-message.alert-error {
     &:before {
-      content: "\e609";
+      content: "\e61c";
```

Comment:
i'm wondering if we should set these values into a variable so moving forward we only have to fix them in one place per icon